<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PasswordView</class>
 <widget class="QWidget" name="PasswordView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>548</width>
    <height>312</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="main_layout">
     <item row="4" column="1">
      <layout class="QHBoxLayout" name="password_layout">
       <item>
        <widget class="QLineEdit" name="password">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="clearButtonEnabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="eye">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../../res/icons/eye-outline.svg</normaloff>
           <normalon>../../../res/icons/eye-off-outline.svg</normalon>../../../res/icons/eye-outline.svg</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="copy">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../../res/icons/content-copy.svg</normaloff>../../../res/icons/content-copy.svg</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="category_edit">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_login">
       <property name="text">
        <string>Логин</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="2">
      <layout class="QHBoxLayout" name="buttons_layout">
       <item>
        <widget class="QPushButton" name="erase">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../../res/icons/delete-outline.svg</normaloff>../../../res/icons/delete-outline.svg</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="save">
         <property name="text">
          <string>Сохранить</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="edit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../../res/icons/pencil-outline.svg</normaloff>../../../res/icons/pencil-outline.svg</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_category">
       <property name="text">
        <string>Категория</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_password">
       <property name="text">
        <string>Пароль</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="login_edit">
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <layout class="QHBoxLayout" name="label_layout">
       <item>
        <widget class="QLabel" name="login">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::TextBrowserInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="category">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="app_edit">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_app">
       <property name="text">
        <string>Приложение</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>edit</sender>
   <signal>toggled(bool)</signal>
   <receiver>save</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>536</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>281</x>
     <y>289</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edit</sender>
   <signal>toggled(bool)</signal>
   <receiver>copy</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>536</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>536</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edit</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_category</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>536</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>75</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edit</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_app</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>536</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>80</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edit</sender>
   <signal>toggled(bool)</signal>
   <receiver>login_edit</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>536</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>353</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edit</sender>
   <signal>toggled(bool)</signal>
   <receiver>category</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>536</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>536</x>
     <y>41</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edit</sender>
   <signal>toggled(bool)</signal>
   <receiver>eye</receiver>
   <slot>toggle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>536</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>492</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edit</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_login</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>536</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>98</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edit</sender>
   <signal>toggled(bool)</signal>
   <receiver>login</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>507</x>
     <y>287</y>
    </hint>
    <hint type="destinationlabel">
     <x>92</x>
     <y>41</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edit</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_password</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>536</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>100</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edit</sender>
   <signal>toggled(bool)</signal>
   <receiver>app_edit</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>532</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>475</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edit</sender>
   <signal>toggled(bool)</signal>
   <receiver>category_edit</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>528</x>
     <y>265</y>
    </hint>
    <hint type="destinationlabel">
     <x>475</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
